utils/start_time.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import os
+import time
+import datetime
+import json
+
+async def save_start_time():
+    dict = {
+        "start_time": time.time()
+    }
+
+    json_object = json.dumps(dict, indent = 4)
+
+    base_path = (Path(os.path.abspath(__file__)) /'..' /'..').resolve()
+    path = base_path / 'time.json'
```

Comment:
These two lines are duplicated in `get_start_time` as well, perhaps they should be in a separate function or even just a global in this file.